docs/index.rst

```diff
@@ -17,6 +17,17 @@ that redirects to a static file service.
 Advanced users can configure a search appliance to support "docker search" and
 can setup repository protection using SSL certificates.
 
+Docker pull
+-----------
+
+There is possibility to ``docker pull`` the image from Crane by its tag or manifest digest.
+In case the client is pulling the image by its manifest ``digest`` ensure that it is the same
+digest that is specified in Crane. The manifest digest, which is provided and stored
+during "docker push" to the private/public docker registry, can differ from the one that
+is stored in Crane. It maybe because Pulp syncs and saves manifests with Schemav1, but
+docker-daemon >= 1.10 is configured to automatically to receive Schemav2(if available), this
+behaviour may lead to the situation where the manifest would have a different digest.
+
```

Comment:
Perhaps this is a better way to say it: 

Manifests generated using docker >= 1.10 use schema version 2. When Pulp requests a manifest from a registry, it always requests the manifest converted to schema version 1. This means that the original manifest digest stored in the registry is different from the manifest digest stored in Pulp. Crane users pulling images by digest, should use the digest stored in Pulp, not the digest stored in the registry.